### PR TITLE
Added Absolute/Relative copy Path,

### PR DESCRIPTION
**Pull request by Jerrody**

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -549,6 +549,28 @@ dependencies = [
...
+
+[[package]]
+name = "clipboard-win"
+version = "2.2.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "e3a093d6fed558e5fe24c3dfc85a68bb68f1c824f440d3ba5aca189e2998786b"
+dependencies = [
+ "winapi 0.3.9",
+]
+
```
Comment (panekj) [inline Cargo.lock:573]: Why?

**Review by panekj — CHANGES_REQUESTED** [review]

File: Cargo.lock (hunk 1 above)
Comment (Jerrody) [reply Cargo.lock:573]: Hm? I just added a `clipboard` crate for the write the content to OS's Clipboard.

Comment (panekj) [reply Cargo.lock:573]: We don't need more dependencies, clipboard is already provided via druid
https://github.com/lapce/lapce/blob/d695914acd737083561eac1bb17b916b53214901/lapce-data/src/document.rs#L58

Comment (Jerrody) [reply Cargo.lock:573]: > We don't need more dependencies, clipboard is already provided via druid
> 
> https://github.com/lapce/lapce/blob/d695914acd737083561eac1bb17b916b53214901/lapce-data/src/document.rs#L58

Oh, okay. Didn't know. Okay, I will fix this.

File: lapce-data/src/config.rs [hunk 2]
```diff
@@ -168,7 +168,7 @@ pub struct EditorConfig {
     #[field_names(
         desc = "Set the editor line height. If less than 5.0, line height will be a multiple of the font size."
     )]
-    line_height: f64,
+    pub line_height: f64,
```
Comment (panekj) [inline lapce-data/src/config.rs:171]: Doesn't have to be public

**Review by panekj — CHANGES_REQUESTED** [review]

File: lapce-data/src/config.rs (hunk 2 above)
Comment (Jerrody) [reply lapce-data/src/config.rs:171]: > Doesn't have to be public

At this case I get a compilation error.

Comment (panekj) [reply lapce-data/src/config.rs:171]: What error? I've just built HEAD without any issues

Comment (Jerrody) [reply lapce-data/src/config.rs:171]: > What error? I've just built HEAD without any issues

I don't know. For the first time I got a compilation error. Anyway, now everything is ok!

**Pull request closed without merging**